EFCache.Redis/RedisCache.cs

```diff
@@ -148,19 +147,19 @@ public bool GetItem(string key, out object value, DbConnection backingConnection
 			dependentEntitySets.GuardAgainstNull(nameof(dependentEntitySets));
 
 			var database = Connection.GetDatabase();
-
-			key = HashKey(key);
+			var hashedKey = HashKey(key);
```

Comment:
I might stick with just `key` here, since it's not really a "hashed key" but a "key that is a hash of a query". As a suggestion.

Reply:
The only thing I changed here was not overwriting the incoming parameter, by instead using a local var. Also this naming is consistent with the other methods.